pages/partner-with-us.js

```diff
@@ -1,12 +1,84 @@
+import { BLOCKS } from '@contentful/rich-text-types';
 import React from 'react';
+import { createClient } from 'contentful';
+import { documentToReactComponents } from '@contentful/rich-text-react-renderer';
+
+import Card, {
+  CardAddress,
+  CardParagraph,
+  CardTitle,
+} from '../components/Card';
 
-import ElectronicsDropOffCard from '../components/ElectronicsDropOffCard';
 import PrimaryLayout from '../components/PrimaryLayout';
 
-export default function PartnerWithUs() {
+const options = {
+  renderNode: {
+    [BLOCKS.PARAGRAPH]: (_, children) => (
+      <CardParagraph>{children}</CardParagraph>
+    ),
+    [BLOCKS.UL_LIST]: (_, children) => (
+      <ul className="list-disc text-left pl-6">
+        {children.map((item) => (
+          <li key={item.key}>{item.props.children[0].props.children[0]}</li>
+        ))}
+      </ul>
+    ),
+  },
+};
+
+export async function getServerSideProps() {
+  const client = createClient({
+    environment: process.env.CONTENTFUL_ENVIRONMENT,
+    space: process.env.CONTENTFUL_SPACE_ID,
+    accessToken: process.env.CONTENTFUL_ACCESS_TOKEN,
+  });
+
+  const res = await client.getEntries({
+    content_type: 'pageWithPartnerCards',
+    'fields.slug': 'partnerwithuscards',
+  });
+
+  return {
+    props: {
+      page: res.items[0],
+    },
+  };
+}
+
+export default function PartnerWithUs({
+  page: {
+    fields: { partnerCards },
+  },
+}) {
   return (
     <PrimaryLayout>
-      <ElectronicsDropOffCard />
+      {partnerCards.map((partnerCard) => {
+        const {
+          fields: {
+            image: {
+              fields: {
+                file: { url },
+              },
+            },
+            description: rawRichTextField,
+            email: emailRichTextFeild,
```

Comment:
I think it would be better to use just `Text` type for email, not the `Rich Text`

Reply:
```suggestion
            contactInstructions: contactInstructionsRichTextField,
```

Reply:
I made a suggestion but am reconsidering, perhaps even just call the field `contactInstructions` or `contactInstructionsRTF` and rename it in Contentful to match?

I sort of like `contactInstructions` rather than `contactInfo` since it's more explicit about instructions and allows for a bit more flexible description, e.g. "Press the buzzer to be let in"

Reply:
What I meant is that in contentful we do now RichText for that field - email is just a simple text.

Reply:
hi @marekrozmus, It seems like hyperlinks really work well with Richtext Field I was unable to find a good way to integrate mailto: functionality with the long text field.

Reply:
@ozer619 Could keep in contentful only email address and add the `mailto:` stuff on the code side? :)